elasticsearch/json-search-1plus.yml

```diff
@@ -130,6 +167,43 @@ observation:
         description: fetch latency on all (primary and replica) shards
         unit: ms
 
+      - name: suggest.count.total
+        source: suggest_total
+        type: counter
+        agentAggregation: SUM
+        label: suggest count (all)
+        description: suggest count on all (primary and replica) shards
+
+      - name: suggest.latency.time.total
+        source: suggest_time_in_millis
+        type: counter
+        agentAggregation: SUM
+        label: suggest latency (all)
+        description: suggest latency on all (primary and replica) shards
+        unit: ms
+
+      - name: scroll.count.total
+        source: scroll_total
+        type: counter
+        agentAggregation: SUM
+        label: scroll count (all)
+        description: scroll count on all (primary and replica) shards
+
+      - name: scroll.time.total
```

Comment:
Missing `latency` in name to match other similar metrics?

Reply:
Yes, I've changed that now.